entries/iteration.qmd

```diff
@@ -22,22 +22,24 @@ Based on these inspirations, an iteration for us follows this general sequence o
-    - Each team member self-assigns to a task (self-assign to Issue) and estimates the time spent on the tasks by using the Iteration's Roadmap Board. 
-    - We'll review whether the number of tasks, time estimation of those tasks, and the aim of the iteration are realistic in the time available.
-    - We'll discuss and agree on how long the iteration should be or will take and set a date for the iteration's last meeting.
+    - During the meeting, we'll discuss and decide or agree on the end output (the "increment" or "milestone").
+    - During the meeting, we'll brainstorm and write out more tasks that are needed to complete the increment or milestone, or select them from the existing list of tasks. Preferably someone will share their screen (if virtual) and the team goes through the tasks together.
+    - While making tasks (Issues), keep them as small as is reasonable and as descriptive and targeted as possible.
+    - As we discuss each task, we collectively assign team members to tasks (assign to the Issues).
+    - We'll review the number of tasks and the aim of the iteration, then discuss and agree on how long the iteration should be (can range from 2-6 weeks, but prefer shorter iterations over longer ones).
+    - After the iteration length is decided, we'll schedule an approximately 2 hour debrief/retrospective meeting on the end date.
+    - Either during or afterward the planning meeting, each team member individually estimates how much time they might need on their assigned tasks, taking into consideration how much time they have available during their work on the iteration. They'll use the the Iteration's Roadmap Board to track and estimate their time spent on tasks.
 3. Throughout the iteration, we will:
     - Have our [update meetings](update-meetings.md) to discuss progress, next steps, and any struggles or barriers (with the work or the iteration/process).
     - Work on our assigned tasks.
     - Review Pull Requests from others.
     - Add more Issues if required. If the Issue is relevant to the current iteration, then add it to the Project Board so that we can determine who will self-assign it during one of our meetings. If the Issue isn't relevant, don't add it to the Project Board, and we will save it for future iterations to work on. 
     - Comment on Issues or Pull Requests if tagged or if we have something to add to the discussion.
-4. The iteration's last meeting is a debrief to discuss the how things went. These are between 1-2 hours and are preferably in person.
-    - How things went and what could be improved in the process.
-    - Barriers encountered that we should address for next time.
+4. The iteration's last meeting is a debrief to discuss how things went. These are between 1-2 hours and are preferably in person. It will have been scheduled during the planning meeting. 
```

Comment:
Ah, so we should have scheduled one for the end of the current iteration at the meeting today then?